### PR TITLE
refactor(openapi): add docs for status messages

### PR DESCRIPTION
**Pull request by kajm**
## Explain the changes you’ve made
Added openapi documentation for status api

## Explain why these changes are made
We want our api:s to be documented, hence creating this openapi docs.

## How to test

1. Checkout this branch
2. Open status-api with VsCode
3. runt the openapi.yml file in swagger

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3xcre5g Skapa OpenAPI för status service](https://app.clickup.com/t/3xcre5g)

File: services/status-api/openapi.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+paths:
+  /status/messages:
+    get:
+      tags:
+        - Status
+      summary: Get API status messages.
+      description: |
+        This endpoint returns a list of messages that would be used and displayed by a consumer.
+        The list of messages returned in the response could be on of the following:
```
Comment (MasterKenth) [inline services/status-api/openapi.yml:25]: Should refrain from using suggestive actions I think, so skip the "that would be used..." part and simply just explain how this endpoint works (inputs and outputs).

Could also probably rephrase this to something like "in order of severity:".

File: services/status-api/openapi.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        1. info
+        2. warning
+        3. maintenance
+
+        in which the severity of the message is increased with start from position1.
+
+      security:
+        - ApiKeyAuth: []
+      parameters:
```
Comment (MasterKenth) [inline services/status-api/openapi.yml:35]: This endpoint does not require a JWT (only api key).

**Review by MasterKenth — CHANGES_REQUESTED** [review]
comments.

**Review by MasterKenth — APPROVED** [review]
